ui/components/providers/workflow/forms/base-credentials-form.tsx

```diff
@@ -25,9 +26,19 @@ import { AWSRoleCredentialsForm } from "./select-credentials-type/aws/credential
 import { GCPDefaultCredentialsForm } from "./select-credentials-type/gcp/credentials-type";
 import { GCPServiceAccountKeyForm } from "./select-credentials-type/gcp/credentials-type/gcp-service-account-key-form";
 import { AzureCredentialsForm } from "./via-credentials/azure-credentials-form";
+import { GitHubCredentialsForm } from "./via-credentials/github-credentials-form";
 import { KubernetesCredentialsForm } from "./via-credentials/k8s-credentials-form";
 import { M365CredentialsForm } from "./via-credentials/m365-credentials-form";
 
+const VIA_VALUES_WITH_BACK_BUTTON = [
```

Comment:
could you clarify what this piece of code does?

Reply:
Can you answer this @alejandrobailo, please? I think you added that part.